analyzer/CommandLine.Analyzer/CommandLine.Analyzer.csproj

```diff
@@ -1,22 +1,15 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netstandard1.3</TargetFramework>
     <IncludeBuildOutput>false</IncludeBuildOutput>
-    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
-  </PropertyGroup>
-  
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
These control where the output of the analyzer goes and that is used during NuGet package creation. But I see that you included that directly in the project for generating the package.

What about the layout of the files on disk during build? I assume that is also impacted by this change as the files will not land under the bin folder for each build project.

Reply:
Yes it will be under bin\Release\netstandard1.3 folder,  I don't see the need to use non standard path to make the package

Reply:
I don't mind that much how the package is being created -- but do like to have all the binaries in a single location. I find that very useful when trying to find the binaries on disk.

Anyway, I can fix this after this is merged.